pkg/kubectl/cmd/config/view.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
+	clientcmdapi "github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd/api"
+)
+
+type viewOptions struct {
+	pathOptions *pathOptions
+	merge       bool
+}
+
+func NewCmdConfigView(out io.Writer, pathOptions *pathOptions, clientConfig clientcmd.ClientConfig) *cobra.Command {
+	options := &viewOptions{pathOptions: pathOptions}
+
+	cmd := &cobra.Command{
+		Use:   "view",
+		Short: "displays the specified .kubeconfig file or a merged result",
```

Comment:
s/specified/current/, since the user isn't telling us what .kubeconfig is currently in use.